ceph.spec.in

```diff
@@ -3,6 +3,7 @@
 %bcond_with tests
 %bcond_without tcmalloc
 %bcond_without libs_compat
+%bcond_with librocksdb
```

Comment:
@liewegas do you have time to comment, as this and more importantly https://github.com/ceph/ceph/pull/5106 which depends on this could both go in and are both relatively trivial, if this issue is allowed to be conditionalized.